.env.sample

```diff
@@ -3,6 +3,8 @@ NAME=metabase
 HOST=local.buildingenvelopedata.org
 HTTP_PORT=4040
 HTTPS_PORT=4041
+BOOTSTRAP_USER_PASSWORD=abcABC123@
+OPEN_ID_CONNECT_CLIENT_SECRET=secret
```

Comment:
@christoph-maurer @shreyaponnamma04 You need to add these two lines two your `.env` file in the project root. You can do it right now. You must do it once you work on the latest version of the `develop` branch.

Reply:
Thank you, @simon-wacker !